style.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-color: #1c1a19;
+  background-image: url("Group.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-top: 120px;
+  padding-bottom: 120px;
+  width: 375px;
+  height: 590px;
+}
```

Comment:
I think it's better to use classes than HTML tags. CSS Classes will help you stylize HTML elements quickly.